packages/property-grid/src/components/PropertyGridUiItemsProvider.tsx

```diff
@@ -0,0 +1,47 @@
+import { connect } from "react-redux";
+
+import { I18N } from "@bentley/imodeljs-i18n";
+import {
+  AbstractWidgetProps,
+  StagePanelLocation,
+  StagePanelSection,
+  UiItemsProvider,
+} from "@bentley/ui-abstract";
+import { FrameworkState } from "@bentley/ui-framework";
+
+import { PropertyGrid, PropertyGridProps } from "./PropertyGrid";
+
+import React = require("react");
```

Comment:
any reason this is being imported with a require?

Reply:
No reason, it was done this way by the "add auto import". I'll fix.

Reply:
fixed